manim/utils/paths.py

```diff
@@ -16,58 +18,361 @@
 
 STRAIGHT_PATH_THRESHOLD = 0.01
 
+PATH_FUNC_TYPE = Callable[[np.ndarray, np.ndarray, float], np.ndarray]
```

Comment:
Curious: I realize there are a few methods that when passed need not be called, as in the parenthesis pair after the method. I wonder if it would be helpful if that was mentioned in a note admonition at the top of the page, what do you think?

Reply:
I think the only function that cannot be called is `straight_path`. I think it would be better to make it consistent with other functions, instead of adding this note.

Reply:
Done